keyboards/orthodox/keymaps/drashna/config.h

```diff
@@ -34,6 +34,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 
 /* key combination for magic key command */
+#undef IS_COMMAND
```

Comment:
The keyboard doesn't have `#ifndef` around its `#define IS_COMMAND`?

Reply:
Nope, and no other board has. 

And I think this should be fine, to be honest.